File numpy-9999/.ipc_in is a fifo while file numpy-9999.new/.ipc_in is a fifo
File numpy-9999/.ipc_out is a fifo while file numpy-9999.new/.ipc_out is a fifo
diff -urN numpy-9999/work/numpy-9999/numpy/distutils/system_info.py numpy-9999.new/work/numpy-9999/numpy/distutils/system_info.py
--- numpy-9999/work/numpy-9999/numpy/distutils/system_info.py	2019-10-06 18:59:16.890968192 -0400
+++ numpy-9999.new/work/numpy-9999/numpy/distutils/system_info.py	2019-10-06 19:03:50.132029357 -0400
@@ -374,30 +374,7 @@
       1 - display warning message
       2 - raise error
     """
-    cl = {'atlas': atlas_info,  # use lapack_opt or blas_opt instead
-          'atlas_threads': atlas_threads_info,                # ditto
-          'atlas_blas': atlas_blas_info,
-          'atlas_blas_threads': atlas_blas_threads_info,
-          'lapack_atlas': lapack_atlas_info,  # use lapack_opt instead
-          'lapack_atlas_threads': lapack_atlas_threads_info,  # ditto
-          'atlas_3_10': atlas_3_10_info,  # use lapack_opt or blas_opt instead
-          'atlas_3_10_threads': atlas_3_10_threads_info,                # ditto
-          'atlas_3_10_blas': atlas_3_10_blas_info,
-          'atlas_3_10_blas_threads': atlas_3_10_blas_threads_info,
-          'lapack_atlas_3_10': lapack_atlas_3_10_info,  # use lapack_opt instead
-          'lapack_atlas_3_10_threads': lapack_atlas_3_10_threads_info,  # ditto
-          'flame': flame_info,          # use lapack_opt instead
-          'mkl': mkl_info,
-          # openblas which may or may not have embedded lapack
-          'openblas': openblas_info,          # use blas_opt instead
-          # openblas with embedded lapack
-          'openblas_lapack': openblas_lapack_info, # use blas_opt instead
-          'openblas_clapack': openblas_clapack_info, # use blas_opt instead
-          'blis': blis_info,                  # use blas_opt instead
-          'lapack_mkl': lapack_mkl_info,      # use lapack_opt instead
-          'blas_mkl': blas_mkl_info,          # use blas_opt instead
-          'accelerate': accelerate_info,      # use blas_opt instead
-          'x11': x11_info,
+    cl = {'x11': x11_info,
           'fft_opt': fft_opt_info,
           'fftw': fftw_info,
           'fftw2': fftw2_info,
